Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -4182,40 +4182,45 @@ final class SafeDIToolTests: XCTestCase {
         await assertThrowsError(
             """
             Dependency cycle detected!
-            DefaultNetworkService -> DefaultLoggingService -> DefaultNetworkService
```

Comment:
this two-element cycle made it very difficult to determine whether the ordering of types in our error message was correct.